src/generators/java/renderers/ClassRenderer.ts

```diff
@@ -10,6 +10,7 @@ import { FormatHelpers } from '../../../helpers';
 import { JavaOptions } from '../JavaGenerator';
 import { ClassPresetType } from '../JavaPreset';
 import { unionIncludesBuiltInTypes } from '../JavaConstrainer';
+import { isEnum } from '../../csharp/Constants';
```

Comment:
Think it's time to move this to general :laughing:

Reply:
Yes 😄